android/java/org/chromium/chrome/browser/util/ImageUtils.java

```diff
@@ -99,4 +109,46 @@ public static Bitmap addGradient(Bitmap src) {
 
         return result;
     }
-}
\ No newline at end of file
+
+    public static void loadSearchEngineLogo(
```

Comment:
@stoletheminerals i am using the same function for load logos as chromium in SearchEnginesAdapter.

Reply:
https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/search_engines/android/java/src/org/chromium/chrome/browser/search_engines/settings/SearchEngineAdapter.java;drc=17b1d05d2ccda19c3ebd903075227bc8e851acf0;l=497

Reply:
That's "fine". Resolving, since there is no Rule of 2 violations, but there is still something we need to fix - https://github.com/brave/reviews/issues/1784#issuecomment-2457865042